swap_meet/item.py

```diff
@@ -0,0 +1,28 @@
+class Item:
+    def __init__(self, age = 0, condition = 0, category = None):
+        if category == None:
+            self.category = ""
+        else:
+            self.category = category
```

Comment:
You only need to use the `None` trick if the structure is mutable.  Luckily Strings in Python are immutable:

```suggestion
    def __init__(self, age = 0, condition = 0, category = ""
        self.category = category
```